src/dehydrated_devices.rs

```diff
@@ -21,6 +21,51 @@ impl From<dehydrated_devices::DehydratedDevices> for DehydratedDevices {
     }
 }
 
+/// Dehydrated device key
+#[wasm_bindgen]
+#[derive(Debug)]
+pub struct DehydratedDeviceKey {
+    pub(crate) inner: Uint8Array,
+}
+
+#[wasm_bindgen]
+impl DehydratedDeviceKey {
+    /// Generates a new random pickle key.
```

Comment:
I'm a bit confused about why we refer to it as a "pickle key" here (and in the other methods), but the type (and documentation on the type itself) call it a "dehydrated device key".

I guess a "pickle key" and a "dehydrated device key" are the same thing, somehow? Basically it feels like the documentation could do with some clarification. It might be clearer if we stuck with "dehydrated device key" throughout.

Reply:
Using `dehydrated device key` everywhere now.
I'll have to do something similar in sdk and ffi